lesson_04/expression/expression_app/src/main/java/com/codedifferently/lesson4/ExpressionCalculator.java

```diff
@@ -1,15 +1,35 @@
 package com.codedifferently.lesson4;
 
 public class ExpressionCalculator {
-  /**
-   * Returns the calculation of ((a + b) * c) / d^e
-   */
-  public double calculate(double a, double b, double c, double d, double e) {
-    // Implement your code here to return the correct value.
-    return 0d;
+  // /**
```

Comment:
This comment is in the wrong place and is also out of date. Can you add the comment back to the calculate method? Thanks!